contracts/Cargo.toml

```diff
@@ -17,6 +17,7 @@ mini-alloc.workspace = true
 cfg-if = "1.0"
 
 [dev-dependencies]
+alloy-primitives = { version = "0.3.1", features = ["arbitrary"] }
```

Comment:
I believe alloy is configured in main `Cargo.toml`

Reply:
It was used without arbitrary feature. Since we're using resolver = 2 we can plugin features necessary just for testing purposes